hist/hist/src/TFormula.cxx

```diff
@@ -3236,6 +3242,107 @@ void TFormula::GradientPar(const Double_t *x, Double_t *result)
    }
 }
 
+/// returns true on success.
+bool TFormula::GenerateHessianPar()
+{
+   // We already have generated the hessian.
+   if (fHessMethod)
+      return true;
+
+   if (!HasHessianGenerationFailed()) {
+      // FIXME: Move this elsewhere
```

Comment:
Now is a good time to factor this out and do what the FIXME suggests :-) Please refactor this rather than just duplicating the code.

Reply:
Currently the PR is very barebones and doesn't work with `clad` either. Once I have added support for arrays in `clad::hessian` I will work on this to completion.